doc/quickstart/ETF_quickstart.rst

```diff
@@ -0,0 +1,286 @@
+:Author: Clemens Portele
+:Author: Jon Herrmann
+:Author: Marco Minghini
+:Author: Enrique Soriano
+:License: Creative Commons Attribution (cc-by) 4.0
+:Thanks: 
+
+@LOGO_ETF@
+@OSGEO_KIND_ETF@
+@VMDK_ETF@
+
+
+
+.. |GS| replace:: GeoServer
+.. @NAME_ETF@ replace::ETF
+
+
+********************************************************************************
+@NAME_ETF@ Quickstart
+********************************************************************************
+
+ETF is an open source testing framework for validating spatial data, metadata and web services in Spatial Data Infrastructures (SDIs). The design of ETF is driven by three goals: be user-friendly, consistent with the standards and capable of testing all resources in an SDI.
```

Comment:
Not urgent:
There are lines that are way over 120 characters. (kinda these three lines is one line)
More or less this is how I see the file:
https://raw.githubusercontent.com/OSGeo/OSGeoLive-doc/4209588cf4a43cd148e07941e162d633ef09fb5d/doc/quickstart/ETF_quickstart.rst
(Maybe your editor gives you the impression that is changing lines)

When there are little issues remaining for me to fix (issues are because of links or sphinx formatting) then
I re-format those long lines. and make them 80 chars long.
No issues, I don't touch a thing.

I reiterate, its not urgent, more urgent is to have the overview on the same PR.
The last thing I worry about is formatting. but is worth to mention.